src/main/kotlin/lotto/domain/LottoRank.kt

```diff
@@ -0,0 +1,15 @@
+package lotto.domain
+
+enum class LottoRank(
+    val sameCount: Int,
+    val amount: Int
+) {
+    FIRST_PLACE(6, 2000000000),
```

Comment:
```suggestion
    FIRST_PLACE(6, 2_000_000_000),
```
이런식으로 가독성을 높힐 수 있습니다 ㅎ